sirepo/pkcli/madx.py

```diff
@@ -63,7 +63,6 @@ def _generate_ptc_particles_file(run_dir, data, twiss):
     if 'bunchReport' in data.report:
         v.summaryData = twiss
         simulation_db.write_json(run_dir.join(template.BUNCH_PARTICLES_FILE), v)
-        return
```

Comment:
was this intentional?

Reply:
Yes, both data files are needed, one for json, one for data download.